sdk/extensions/Azure.AspNetCore.DataProtection.Blobs/tests/DataProtectionBlobFunctionalTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Storage;
+using Azure.Storage.Blobs;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+
+namespace Azure.AspNetCore.DataProtection.Blobs.Tests
+{
+    public class DataProtectionBlobFunctionalTest
```

Comment:
Nit: class names typically end in "Tests". Might aide in discoverability when using class name filters down the road.